packages/gasket-plugin-config/README.md

```diff
@@ -169,6 +169,26 @@ module.exports = {
 };
 ```
 
+### Config with Public config
+
+If you are not using Redux, but still need access to config values in client-side code, you can define a `public` property in your `gasket.config.js`.
+
+```js
+module.exports = {
+  public: {
+    test1: 'config value 1 here',
+    test2: 'config value 2 here'
+  }
+};
+```
+The config plugin will return these `public` properties to your browser, to be accessed by the `@gasket/data` plugin, and used like so:
```

Comment:
Refer to `@gasket/data` as a package or module, to distinguish it from a Gasket plugin (with lifecycle hooks, etc)
```suggestion
The config plugin will return these `public` properties to your browser, to be accessed by `@gasket/data`, and used like so:
```

Reply:
Also, describe that what's placed in `public` in the config files, is accessible as `.config` from gasketData. You show that in the example, but should describe it.